mmv1/products/datastore/Index.yaml

```diff
@@ -28,7 +28,6 @@ description: |
 id_format: 'projects/{{project}}/indexes/{{index_id}}'
 error_retry_predicates: ['transport_tpg.DatastoreIndex409Contention']
 autogen_async: true
-skip_sweeper: true
```

Comment:
why was this skipeed in the first place? It just works?

Reply:
I'm not sure why it was skipped. When I run it locally this will make a request to `https://datastore.googleapis.com/v1/projects/{project}/indexes`, and returns project does not exist, which is expected in my case.